stable/fairwinds-insights/README.md

```diff
@@ -110,6 +110,10 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | closeTicketsCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"2Gi"},"requests":{"cpu":"500m","memory":"1.5Gi"}}` | Resources for the close tickets job. |
 | closeTicketsCronjob.schedules | list | `[{"cron":"0/15 * * * *","name":"close-tickets"}]` | CRON schedules for the close tickets job. |
 | closeTicketsCronjob.securityContext.runAsUser | int | `10324` | The user ID to run the close tickets job under. |
+| cloudCostsUpdateCronjob.enabled | bool | `true` | Cloud costs update enabled by default |
+| cloudCostsUpdateCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"2Gi"},"requests":{"cpu":"500m","memory":"1.5Gi"}}` | Resources for the cloud costs update job. |
```

Comment:
Update docs: actual `requests.memory` is 2Gi according to `values.yaml`

Reply:
Fixed